Runtime/Scripts/ExampleWebSocket.cs

```diff
@@ -54,7 +55,10 @@ async Task StartWebSocket(string url)
             await wsClient.Connect(uri);
         }
 
-
+        private void HandleFindCloudlet(EdgeEventsStatus status, FindCloudletEvent fcEvent)
+        {
+            print("NewFindCloudlet triggered status is  " + status.status + ", Trigger" + fcEvent.trigger);
```

Comment:
Maybe log the base AppInst FQDN here (But not the prefix(es)).

Reply:
Done